src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -354,7 +354,7 @@ protected async Task<bool> SettingsFileHasChangedAsync()
         {
             string fileName = await GetLaunchSettingsFilePathAsync();
 
-            return !_fileSystem.FileExists(fileName) || _fileSystem.LastFileWriteTime(fileName) != LastSettingsFileSyncTime;
+            return !_fileSystem.FileExists(fileName) || _fileSystem.LastFileWriteTimeUtc(fileName) != LastSettingsFileSyncTime;
```

Comment:
Consider renaming `LastSettingsFileSyncTime` to `LastSettingsFileSyncTimeUtc` for clarity.